core/src/ledger/governance/utils.rs

```diff
@@ -226,14 +261,17 @@ pub fn compute_proposal_result(
 ) -> ProposalResult {
     let mut yay_voting_power = VotePower::default();
     let mut nay_voting_power = VotePower::default();
+    let mut abstain_voting_power = VotePower::default();
 
     for (address, vote_power) in votes.validator_voting_power {
         let vote_type = votes.validators_vote.get(&address);
         if let Some(vote) = vote_type {
             if vote.is_yay() {
                 yay_voting_power += vote_power;
-            } else {
+            } else if vote.is_nay() {
                 nay_voting_power += vote_power;
+            } else {
```

Comment:
`vote.is_abstain()` would be more defensive, in case we add another voting option in the future